### PR TITLE
change config file name from `.env` to `env` (iss. #16)

### LINKED ISSUE #16: support multiple `.env` files
While walking @ziegler1960 through the process of running the app, it became apparent that he would need various configurations of the `.env` file at any given time.  Reasons for this include, but are not limited to…

* Connecting to various Canvas instances
* Authenticating to the Canvas API with various tokens (keys)
* Working with various Canvas subaccounts
* Working with various Canvas terms
* Working with various LTI tools

Editing the `.env` file each time a different configuration is needed is very inconvenient.

A possible solution would be to keep various versions of `.env` around and copy the one needed into the main one before running the app.  I.e., `cp .env-special-config-1 .env`.  However, that can become awkward and easy to forget which configuration `.env` currently contains.

A better solution would be to include the name of the configuration file to be used as an argument to the app on the command line.  I.e., the command `python migration/main.py .env-special-config-1` would run the app which would read the specified config file instead of `.env`.  If the app is run without an argument, then it would use the `.env` file by default, as it does now.

### PR DESCRIPTION
**Pull request by lsloan**
Changing config file name from `.env` to `env` will make it easier for people with limited shell experience to work with the file.  Hidden files can be tricky for people to work with if they've never done it before.

Partially resolves #16.

No code review required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,27 +4,30 @@ This application migrates LTI tools in a Canvas instance from one version to ano
...
 
 ```shell
-open -a TextEdit .env
+open -a TextEdit env
 ```
 
-Note: Starting the editor this way is recommended because the hidden file `.env` is not visible in the editor's "Open" dialog.
+Carefully fill in the values for each of the variables.  The values must be surrounded by quotes.  E.g.,
+
+```shell
```
Comment (zqian) [inline README.md:29]: Should a sample variable setting be listed here?

**Review by zqian — APPROVED** [review]
Approved with a minor comment.

File: README.md (hunk 1 above)
Comment (lsloan) [reply README.md:29]: That's not a bad idea.

**Comment by lsloan** [comment]
I wanted to get this done today. Now the power is out at home, so I'm committing this even though it may be a little rough.

**Pull request merged**